src/Microsoft.ML.CodeGen/Microsoft.ML.CodeGenerator.csproj

```diff
@@ -0,0 +1,114 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <LangVersion>7.3</LangVersion>
+    <PackageId>Microsoft.ML.CodeGenerator</PackageId>
+    <RootNamespace>Microsoft.ML.CodeGenerator</RootNamespace>
```

Comment:
This is not needed, I understand it's a copy from somewhere but I had to remove these when merging features/automl code into master

Reply:
removed